src/vos/vos_gc.c

```diff
@@ -571,16 +672,33 @@ gc_add_item(struct vos_pool *pool, enum vos_gc_type type, umem_off_t item_off,
 	}
 }
 
+struct vos_container *
+gc_get_container(struct vos_pool *pool)
+{
+	struct vos_container	*cont;
+	/** In order to be fair to other containers, we remove this from the
+	 * list.  If we run out of credits, we will put it at the back of
+	 * the list and give another container a turn next time.
+	 */
+	cont = d_list_pop_entry(&pool->vp_gc_cont, struct vos_container,
+				vc_gc_link);
```

Comment:
What if a container is destroyed? The vos_container will be freed but still linked?

Instead of passing 'coh' to GC, Is there any way to make DTX to skip the entries belong to deleted objects?

Reply:
I see, thanks.